keyboards/buildakb/mw60/readme.md

```diff
@@ -0,0 +1,24 @@
+# MW60 PCB
+
+![MW60 PCB](https://i.imgur.com/5E48mAPh.jpg)
+
+A simple standard 60% PCB available in hotswap and solderable variants
+
+-   Keyboard Maintainer: [Maelkk](https://github.com/Aeonstrife)
+-   Hardware Supported: MW60 PCB
+-   Hardware Availability: [sandkeys.me](https://sandkeys.me/product/petals-60-addon-pcbs/)
```

Comment:
looking at the PCB images from the _Hardware Availability_ link, could you confirm the following key positions are available on the MW60?:
- 4th/Shift Row: 1u-1.75u Right Shift (as seen on ABNT2 and JIS layouts)
```
┌───┬──────┐
│1u │1.75u │
└───┴──────┘
```
- 5th/Bottom Row: _Infinity_ or _Hacker_
```
┌─────┬───┬─────┬───────────────────────┬─────┬───┬───┬─────┐
│1.5u │1u │1.5u │6u                     │1.5u │1u │1u │1.5u │
└─────┴───┴─────┴───────────────────────┴─────┴───┴───┴─────┘
```

Reply:
Hello @dunk2k 
- 4th/Shift Row: 1u-1.75u Right Shift (as seen on ABNT2 and JIS layouts)  
- 5th/Bottom Row: Infinity or Hacker  
both are not available on the MW60

Reply:
Thanks for confirming, and adding Community Layout support ☺️